power-pages-docs/admin/admin-center-analytics.md

```diff
@@ -31,22 +32,22 @@ To access the analytics dashboard, you should have one of the following roles:
 - Global Administrator
 - Dynamics 365 administrator
 - Power Platform administrator
-- System administrator (you can only see environments for which you're a system administrator)
+- System administrator (You can only see environments for which you're a system administrator.)
 
-## View traffic analytics for all websites in a tenant
+### View traffic analytics for all websites in a tenant
 
 1. Sign in to the [Power Platform admin center](https://admin.powerplatform.microsoft.com/).
 1. On the left pane, select **Resources**, then select **Power Pages sites**.
 1. Go to **Analytics**.
     :::image type="content" source="Media/admin-center-analytics/usage-data.png" alt-text="A screenshot of Power Pages sites inside Power Platform admin center displaying Usage data.":::
 
-From here you can view the traffic that your websites are receiving: DAU (Daily Active Users), WAU (Weekly Active Users) and MAU (Monthly Active Users).
+From here you can view the traffic that your websites are receiving: Daily Active Users (DAU), Weekly Active Users (WAU), and Monthly Active Users (MAU).
 
-All charts show the DAU, WAU and MAU across all the sites in the selected environment. You can select the environment of your interest using the drop-down.
+Each chart displays the DAU, WAU, and MAU for all sites within the selected environment. Select your environment in the upper-right corner of the page.
 
 :::image type="content" source="Media/admin-center-analytics/analytics-environment-dropdown.png" alt-text="A screenshot of the Power Platform admin center showing the Analytics Usage Data's Environment dropdown.":::
 
-Within the selected environment, you can also select a specific site for which you want to see the traffic. Select the Site ID from the drop-down (You can select multiple sites by using ctrl + click) to see traffic for that site.
+Within the selected environment, you can also select a specific site for which you want to see the traffic. Select the Site ID from the list (Multiple sites can be selected by using **Ctrl + Click**) to see traffic for that site.
 
 :::image type="content" source="Media/admin-center-analytics/analytics-site-id.png" alt-text="A screenshot fo the Power Platform admin center showing the Analytics Usage Data's Site ID dropdown.":::
```

Comment:
~A screenshot fo the Power~
Screenshot of the Power

Just to be sure (that "left pane" thing shook me!), I looked up our guidance for structuring alt text. [Accessibility guidelines for images and videos](https://review.learn.microsoft.com/en-us/help/contribute/contribute-accessibility-multimedia?branch=main#alt-text-requirements-for-images) does in fact say to start alt text for screenshots with "Screenshot." Whew, I was right! :D BUT, the rest of the alt text should be tweaked to match the guidance in this section: https://review.learn.microsoft.com/en-us/help/contribute/contribute-accessibility-multimedia?branch=main#screenshots-of-user-interface